### PR TITLE
Remove bloat

### PR DESCRIPTION
**Pull request by AlanGriffiths**

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -72,13 +72,43 @@ parts:
       snapcraftctl set-version $mir_version-snap$recipe_version
       if echo $mir_version | grep -e '+dev' -e '~rc' -q; then snapcraftctl set-grade devel; else snapcraftctl set-grade stable; fi
     stage-packages:
-      - mir-graphics-drivers-desktop
```
Comment (wmww) [inline snap/snapcraft.yaml:75]: Are we *sure* no one is using Mir-on-Wayland?

**Review by wmww — APPROVED** [review]
LGTM
bors delegate+

File: snap/snapcraft.yaml (hunk 1 above)
Comment (AlanGriffiths) [reply snap/snapcraft.yaml:75]: Yes, there is no plug to support that:

```yaml
    plugs:
      - login-session-control
      - x11
```

**Comment by AlanGriffiths** [comment]
bors r=@wmww

**Pull request merged**